### ISSUE TITLE
Tabs using wrong aria-attributes to theire roles

### ISSUE THREAD

**Issue by AK2083** [vmware/clarity#2667]
## Describe the bug

I would like to insert tabs into main area. After checking by using the chrome audit tools, I get followed errors:

![image](https://user-images.githubusercontent.com/3448835/45814979-9fd00200-bcd7-11e8-91f6-3535c2141cf4.png)

which are genereated to:

![image](https://user-images.githubusercontent.com/3448835/45815038-d3129100-bcd7-11e8-9b81-0df034e230f0.png)

## How to reproduce
 ```html
<main>
    <div class="content-container">
      <div class="content-area" style="padding-top:0rem">

        <clr-tabs>
          <clr-tab>
            <button clrTabLink>Sequenzansicht</button>
            <ng-template [(clrIfActive)]="dashboardActive">
              <clr-tab-content>
                1
              </clr-tab-content>
            </ng-template>
          </clr-tab>

          <clr-tab>
            <button clrTabLink>Diskusion</button>
            <ng-template [(clrIfActive)]="managementActive">
              <clr-tab-content>
                2
              </clr-tab-content>
            </ng-template>
          </clr-tab>

          <clr-tab>
            <button clrTabLink>Freigabe</button>
            <ng-template [(clrIfActive)]="cloudActive">
              <clr-tab-content>
                3
              </clr-tab-content>
            </ng-template>
          </clr-tab>

          <clr-tab>
            <button clrTabLink>Information</button>
            <ng-template [(clrIfActive)]="infrastructureActive">
              <clr-tab-content>
                4
              </clr-tab-content>
            </ng-template>
          </clr-tab>
        </clr-tabs>
      </div>
   </div>
</main>
```
## Expected behavior

I expect that the attributes are matching to theire roles.

## Versions

**App**

* Angular: 6.1.0
* Node: 8.10.0
* Clarity: 0.13.1-patch.1

**Device:**

* Type: Lenovo Think Pad
* OS: Windows 10 18.03
* Browser: Chrome 69.0.3497.100

**Comment by gnomeontherun** [comment]
This should be resolved by this week's release, see #2551

**Comment by AK2083** [comment]
You are the best. 👍

**Comment by Shijir** [comment]
Released in [0.13.2](https://github.com/vmware/clarity/releases/tag/v0.13.2).
